django_slack_oauth/models.py

```diff
@@ -5,21 +5,17 @@
 
 from jsonfield import JSONField
 
-__all__ = (
-    'SlackUser',
-)
 
+class SlackOAuthRequest(models.Model):
```

Comment:
I can leave the old model and just move the functionality to a pipeline method to keep backward functionality.